arangod/MMFiles/MMFilesEngine.cpp

```diff
@@ -235,6 +235,10 @@ void MMFilesEngine::validateOptions(std::shared_ptr<options::ProgramOptions>) {}
 void MMFilesEngine::prepare() {
   TRI_ASSERT(&(server().getFeature<EngineSelectorFeature>().engine()) == this);
 
+  LOG_TOPIC("80866", WARN, arangodb::Logger::STARTUP)
+      << "The MMFiles engine is deprecated since ArangoDB 3.6. "
```

Comment:
I would add `and will be removed in future versions.`

Reply:
Yes @Simran-B  did ;)